src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ChannelMemberRepository.cs

```diff
@@ -108,10 +108,18 @@ public async Task<IReadOnlyCollection<ChannelMember>> GetChannelMembersWithMembe
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT *
-                                 FROM ChannelMembers cm
-                                 INNER JOIN Members m ON m.Id = cm.MemberId
-                                 WHERE ChannelId = @channelId";
+                var sqlQuery = $@"
+                    SELECT 
+                        {nameof(ChannelMember.ChannelId)}, 
+                        {nameof(ChannelMember.MemberId)}, 
+                        {nameof(ChannelMember.LastReadMessageId)}, 
+                        {nameof(ChannelMember.IsMuted)}, 
+                        {nameof(ChannelMember.IsPinned)}
```

Comment:
We need to add Members to SELECT as we are using INNER JOIN here.

Reply:
done